specification/search/data-plane/Azure.Search/preview/2021-04-30-Preview/searchindex.json

```diff
@@ -239,6 +239,15 @@
               "name": "SearchOptions"
             }
           },
+          {
+            "name": "semanticConfiguration",
```

Comment:
At least for Java, this will result in a private property called `semanticConfiguration` and getter and setter methods `getSemanticConfiguration` and `setSemanticConfiguration`. For the most part Java code generation leaves the Swagger `name` field untouched and uses it directly as-is.